File: frontend/app/src/main/java/com/example/geoquiz_frontend/data/remote/dtos/koth/PlayerJoinedData.java
package com.example.geoquiz_frontend.data.remote.dtos.koth;

public class PlayerJoinedData {
    private String lobbyId;
    private String playerId;
    private String playerName;
    private int playerLevel;
    private int totalPlayers;

    public String getLobbyId() { return lobbyId; }
    public String getPlayerId() { return playerId; }
    public String getPlayerName() { return playerName; }
    public int getPlayerLevel() { return playerLevel; }
    public int getTotalPlayers() { return totalPlayers; }

}

File: backend/GeoQuiz_backend/GeoQuiz_backend/Application/DTOs/KingOfTheHill/PlayerJoinedData.cs
namespace GeoQuiz_backend.Application.DTOs.KingOfTheHill
{
    public class PlayerJoinedData
    {
        public Guid LobbyId { get; set; }
        public Guid PlayerId { get; set; }
        public string PlayerName { get; set; } = null!;  
        public int PlayerLevel { get; set; } 
        public int TotalPlayers { get; set; }
    }
}
